### PR TITLE
Weiqsun rmd

### PR DESCRIPTION
**Pull request by weiqsun**

### CODE REVIEW

**Comment by weiqsun** [comment]
merge with master

**Comment by weiqsun** [comment]
close issue #2

**Comment by mblack88** [comment]
Hello Neighbor, this branch has some conflicts.

File: weiqsun_rmd_a2.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Ann Arbor, MI
+
+Where UofM is located
+
+### reat place for fun and entertainment
```
Comment (mblack88) [inline weiqsun_rmd_a2.md]: We use to eat at the Old German, maybe it should make the list if it is still open.

**Comment by mblack88** [comment]
:hamburger:

File: weiqsun_rmd_a2.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Where UofM is located
 
-### reat place for fun and entertainment
+### great place for fun and entertainment
```
Comment (weiqsun) [inline weiqsun_rmd_a2.md:5]: take the change from 'reat' to 'great'

**Comment by weiqsun** [comment]
able to merged now

**Comment by weiqsun** [comment]
one more commit

**Pull request merged**